da.json

```diff
@@ -1,49 +1,49 @@
+			"option-readable-line-length": "Læsbar linje længde",
+			"option-readable-line-description": "Begræns maksimum linje længde. Mindre indhold tilpasses skærmen, men længere blokke af tekst er mere læsbare.",
+			"option-strict-line-break": "Strengt linjeskift",
+			"option-strict-line-break-description": "Markdown specifikationerne ignorerer linjeskift i forhåndsvisning. Slå dette fra for at gøre enkelt linjeskift synligt.",
+			"option-show-frontmatter": "Vis indledende tekst",
+			"option-show-frontmatter-description": "Vis indledende tekst som en kodeblok i forhåndsvisning. Slå fra for at skjule den indledende tekst sektion.",
+			"option-auto-pair-brackets": "Auto par kantede parenteser",
+			"option-auto-pair-brackets-description": "Par kantede parenteser og citér automatisk.",
+			"option-auto-pair-markdown": "Auto par Markdown syntaks",
+			"option-auto-pair-markdown-description": "Par automatisk symboler for fed og kursiv tekst.",
+			"option-smart-indent-lists": "Smarte indryknings lister",
+			"option-smart-indent-lists-description": "Sæt indrykning automatisk og placér elementer korrekt..",
+			"option-fold-heading": "Fold overskrift",
+			"option-fold-heading-description": "Folder al indhold ind under en overskrift.",
+			"option-fold-indent": "Fold indrykning",
+			"option-fold-indent-description": "Lader dig folde en del af en indrykning, såsom lister.",
+			"option-default-view-mode": "Standard visningstilstand",
+			"option-default-view-mode-description": "Standard tilstanden en fil åbnes i.",
 			"option-default-view-mode-source": "Editor",
-			"option-default-view-mode-preview": "Preview",
-			"option-show-line-number": "Show line number",
-			"option-show-line-number-description": "Show line number in the gutter.",
-			"option-use-tabs": "Use tabs",
-			"option-use-tabs-description": "Use tabs to indent by pressing the “Tab” key. Turn this off to use spaces.",
-			"option-tab-size": "Tab size",
-			"option-tab-size-description": "Number of spaces a tab is equivalent to.",
-			"option-line-wrap": "Line wrap",
-			"option-line-wrap-description": "Wrap long lines to fit within the window, without having to scroll from left to right.",
-			"option-auto-convert-html": "Auto convert HTML",
-			"option-auto-convert-html-description": "Automatically convert HTML to Markdown when pasting and drag-and-drop from webpages. Use Ctrl/Cmd+Shift+V to paste without conversion.",
-			"option-vim-key-bindings": "Vim key bindings",
-			"option-vim-key-bindings-description": "Let you use Vim key bindings in the source editor."
+			"option-default-view-mode-preview": "Forhåndsvisning",
+			"option-show-line-number": "Vis linjenummer",
+			"option-show-line-number-description": "Vis linjenummer i margen.",
+			"option-use-tabs": "Benyt tabulator",
+			"option-use-tabs-description": "Benyt tabulator til at indrykke ved at trykke på "Tab" tasten. Slå den fra for at benytte mellemrumstasten.",
```

Comment:
Please escape the quotes by using `\"` instead of just `"`, since the quote character would end the string.

Reply:
I have corrected it. Seems to add a second commit in this pull request. Hope this is correct.